frontend/src/components/App/Notifications/index.tsx

```diff
@@ -179,6 +191,18 @@ export default function Notifications() {
     setAnchorEl(null);
   };
 
+  function handleNotificationMarkAllRead() {
+    const massagedNotifications = notifications.map(notification => {
+      notification.seen = true;
+      return notification;
+    });
+    dispatch(setUINotifications(massagedNotifications));
+  }
+
+  function handleNotificationClear() {
+    dispatch(setUINotifications([]));
```

Comment:
This will have the effect I had warned about: upon refreshing the page, we get all the notifications back again. We should instead have a way to hide them, so for the user it looks like they've been deleted, but we need them around in order to know if they should be displayed.